examples/fedprox_example/server.py

```diff
@@ -62,10 +54,6 @@ def main(config: Dict[str, Any], server_address: str) -> None:
         config["local_epochs"],
         config["batch_size"],
         config["n_server_rounds"],
-        config["adaptive_proximal_weight"],
```

Comment:
These weren't used anywhere in the config because they are part of the strategy. So I removed them.